traversal/procedure/ProcedureEdge.java

```diff
@@ -81,11 +81,13 @@
 
     private final int order;
     private final Encoding.Direction.Edge direction;
+    private final int hash;
 
     private ProcedureEdge(VERTEX_FROM from, VERTEX_TO to, int order, Encoding.Direction.Edge direction, String symbol) {
         super(from, to, symbol);
         this.order = order;
```

Comment:
This state needs to be removed once the planner natively spits out an ordering for vertices instead of edges.

cc @dmitrii-ubskii